src/Line.js

```diff
@@ -1,5 +1,20 @@
 export class Line {
   constructor() {
     this.parts = [];
+    this.measures = null;
+    this.tablature = null;
+    this.comment = null;
+  }
+
+  hasTablature() {
+    return this.tablature != null;
```

Comment:
are there any scenarios where `this.tablature == []`is true? In that case this would method and the 2 below would return an unexpected true

Reply:
you are right for `tablature`, but I think it still applies to measures as that naming implies an array?
 looking at it, I wonder why `parts` is initialized with an empty array and `measures` is not 🤔

Reply:
Does empty array equate to null? The goal here is to let the user interrogate the Line to figure out what it contains. I suppose maybe I should initialize the Parts to null as well?